public/app/plugins/datasource/prometheus/querybuilder/shared/QueryOptionGroup.tsx

```diff
@@ -2,42 +2,64 @@ import { css } from '@emotion/css';
 import React from 'react';
 import { useToggle } from 'react-use';
 
-import { GrafanaTheme2 } from '@grafana/data';
+import { getValueFormat, GrafanaTheme2 } from '@grafana/data';
 import { Stack } from '@grafana/experimental';
 import { Icon, useStyles2 } from '@grafana/ui';
+import { QueryStats } from 'app/plugins/datasource/loki/components/types';
+import { LokiDatasource } from 'app/plugins/datasource/loki/datasource';
 
 export interface Props {
   title: string;
   collapsedInfo: string[];
   children: React.ReactNode;
+  queryStats?: QueryStats;
+  datasource?: LokiDatasource;
 }
 
-export function QueryOptionGroup({ title, children, collapsedInfo }: Props) {
+export function QueryOptionGroup({ title, children, collapsedInfo, queryStats, datasource }: Props) {
   const [isOpen, toggleOpen] = useToggle(false);
-  const styles = useStyles2(getStyles);
+
+  const querySizeThresholdInBytes = 50000;
```

Comment:
I would remove all of the code related to `treshold` and implement it once we know what treshold we want to use.